local_devnet/scripts/init.sh

```
@@ -5,21 +5,21 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 $CELESTIA_BIN config set app api.enable true
 $CELESTIA_BIN keys add alice
 
 if [ "$MULTIPLEXER" = "true" ]; then
-cd $SCRIPT_DIR; cp ../celestia-app/genesis_046.json $CELESTIA_HOME/config/genesis.json; cd $PWD # use local_devnet genesis
+cd $SCRIPT_DIR; cp ../celestia-app/genesis_046.json $CELESTIA_APP_HOME/config/genesis.json; cd $PWD # use local_devnet genesis
 $CELESTIA_BIN passthrough v3 add-genesis-account alice 5000000000utia --keyring-backend test
 $CELESTIA_BIN passthrough v3 gentx alice 1000000utia --fees 1utia --chain-id local_devnet
 $CELESTIA_BIN passthrough v3 collect-gentxs
 else
-cd $SCRIPT_DIR; cp ../celestia-app/genesis.json $CELESTIA_HOME/config/genesis.json; cd $PWD # use local_devnet genesis
+cd $SCRIPT_DIR; cp ../celestia-app/genesis.json $CELESTIA_APP_HOME/config/genesis.json; cd $PWD # use local_devnet genesis
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)